packages/fast-usdc/src/update-noble-ica.core.js

```diff
@@ -28,7 +28,7 @@ const makeRatio = (numerator, numeratorBrand, denominator = 100n) =>
  */
 
 /**
- * Update feeConfig.variableRate to 0.5%, i.e. 5n/1000n
+ * Update feeConfig.variableRate to 0.1%, i.e. 1n/1000n
```

Comment:
I wonder if we want to phrase this exactly as the request was made to us:

```suggestion
 * Update feeConfig.variableRate to 10 basis points, i.e. 10n/10_000n
```

or with a `BASIS_POINTS` constant, just:

```suggestion
 * Update feeConfig.variableRate to 10 basis points
```

probably not worth another ci spin